### PR TITLE
[airq] Add health/performance channels in percent, add UoM, refine unit hints and state patterns, make item labels consistent

### PR DESCRIPTION
**Pull request by fwolter**
This PR is based on #16696.

### CODE REVIEW

File: bundles/org.openhab.binding.airq/src/main/resources/OH-INF/thing/thing-types.xml [hunk 1]
```diff
@@ -216,22 +220,34 @@
...
 	<channel-type id="humidity" advanced="false">
 		<item-type unitHint="%">Number:Dimensionless</item-type>
 		<label>Humidity</label>
-		<state readOnly="true" pattern="%.2f %%"></state>
+		<state readOnly="true" pattern="%.2f %unit%"></state>
 	</channel-type>
 
 	<channel-type id="humidity_abs" advanced="false">
-		<item-type>Number</item-type>
+		<item-type unitHint="g/m³">Number:Density</item-type>
```
Comment (J-N-K) [inline bundles/org.openhab.binding.airq/src/main/resources/OH-INF/thing/thing-types.xml:248]: This requires update instructions

File: bundles/org.openhab.binding.airq/src/main/resources/OH-INF/thing/thing-types.xml [hunk 2]
```diff
@@ -241,51 +257,57 @@
 	</channel-type>
 
 	<channel-type id="no2" advanced="false">
-		<item-type>Number</item-type>
-		<label>NO₂ Concentration</label>
-		<state readOnly="true" pattern="%.2f μg/m³"></state>
+		<item-type unitHint="μg/m³">Number:Density</item-type>
```
Comment (J-N-K) [inline bundles/org.openhab.binding.airq/src/main/resources/OH-INF/thing/thing-types.xml:260]: see above

File: bundles/org.openhab.binding.airq/src/main/resources/OH-INF/thing/thing-types.xml [hunk 3]
```diff
@@ -241,51 +257,57 @@
...
+		<label>Nitrogen Dioxide (NO₂)</label>
+		<state readOnly="true" pattern="%.2f %unit%"></state>
 	</channel-type>
 
 	<channel-type id="o3" advanced="false">
-		<item-type>Number</item-type>
-		<label>O₃ Concentration</label>
-		<state readOnly="true" pattern="%.2f μg/m³"></state>
+		<item-type unitHint="µg/m³">Number:Density</item-type>
```
Comment (J-N-K) [inline bundles/org.openhab.binding.airq/src/main/resources/OH-INF/thing/thing-types.xml:266]: see above

File: bundles/org.openhab.binding.airq/src/main/resources/OH-INF/thing/thing-types.xml [hunk 4]
```diff
@@ -241,51 +257,57 @@
...
+		<item-type>Number:Dimensionless</item-type>
+		<label>Performance Index Raw</label>
 		<state readOnly="true"></state>
 	</channel-type>
 
 	<channel-type id="pm1" advanced="false">
-		<item-type>Number</item-type>
-		<label>Fine Dust Concentr. >1μ</label>
-		<state readOnly="true" pattern="%.0f μg/m³"></state>
+		<item-type unitHint="μg/m³">Number:Density</item-type>
```
Comment (J-N-K) [inline bundles/org.openhab.binding.airq/src/main/resources/OH-INF/thing/thing-types.xml:290]: see above

File: bundles/org.openhab.binding.airq/src/main/resources/OH-INF/thing/thing-types.xml [hunk 5]
```diff
@@ -241,51 +257,57 @@
...
+		<label>Fine Dust Concentr. >1μm</label>
+		<state readOnly="true" pattern="%.2f %unit%"></state>
 	</channel-type>
 
 	<channel-type id="pm10" advanced="false">
-		<item-type>Number</item-type>
-		<label>Fine Dust Concentr. >10μ</label>
-		<state readOnly="true" pattern="%.0f μg/m³"></state>
+		<item-type unitHint="μg/m³">Number:Density</item-type>
```
Comment (J-N-K) [inline bundles/org.openhab.binding.airq/src/main/resources/OH-INF/thing/thing-types.xml:296]: see above

File: bundles/org.openhab.binding.airq/src/main/resources/OH-INF/thing/thing-types.xml [hunk 6]
```diff
@@ -241,51 +257,57 @@
...
+		<label>Fine Dust Concentr. >10μm</label>
+		<state readOnly="true" pattern="%.2f %unit%"></state>
 	</channel-type>
 
 	<channel-type id="pm2_5" advanced="false">
-		<item-type>Number</item-type>
-		<label>Fine Dust Concentr. >2,5μ</label>
-		<state readOnly="true" pattern="%.0f μg/m³"></state>
+		<item-type unitHint="μg/m³">Number:Density</item-type>
```
Comment (J-N-K) [inline bundles/org.openhab.binding.airq/src/main/resources/OH-INF/thing/thing-types.xml:302]: see above

**Comment by lsiepel** [comment]
Please rebase now the other PR is merged. You might need to look into the thingtypeversion as version 1 is allready merged. (did not look if they are about the same Thing)

**Comment by fwolter** [comment]
I added the update instructions. Thank you both for pointing that out!

File: bundles/org.openhab.binding.airq/README.md [hunk 7]
```diff
@@ -53,26 +53,28 @@ The rw column is empty if the channel is only readable, w if the channel can be
 | fineDustCnt02_5           | Number:Dimensionless |    | Fine Dust >2,5 µm                                                   |
 | fineDustCnt05             | Number:Dimensionless |    | Fine Dust >5 µm                                                     |
 | fineDustCnt10             | Number:Dimensionless |    | Fine Dust >10 µm                                                    |
-| co                        | Number               |    | CO concentration                                                    |
-| co2                       | Number:Dimensionless |    | CO₂ concentration                                                   |
+| co                        | Number:Density       |    | Carbon monoxide (CO) concentration                                  |
+| co2                       | Number:Dimensionless |    | Carbon dioxide (CO₂) concentration                                  |
```
Comment (lsiepel) [inline bundles/org.openhab.binding.airq/README.md:57]: Why is there a difference? I would expect both to be `Number:Density` ?

The other channels are also diffent while they all show the concentration of something.

Comment (fwolter) [reply bundles/org.openhab.binding.airq/README.md:57]: CO2 is measured in ppm like oxygen is measured in %. CO should always be 0%, maybe that's why it's measured in µg/m³. In any case, the difference is correct.

Comment (lsiepel) [reply bundles/org.openhab.binding.airq/README.md:57]: Thanks for the details, learned something here ;-)

**Review by lsiepel — APPROVED** [review]
Thanks, LGTM

**Pull request merged**